docs/dev/start-guide.md

```diff
@@ -39,3 +41,7 @@ When running integration tests from an IDE the following environment variables n
 `ATLAS_PUBLIC_KEY=<..>`
 `ATLAS_PRIVATE_KEY=<..>`
 `GINKGO_EDITOR_INTEGRATION=true`
+
+# Known issues
+If you get an error `controller-gen: command not found` or `kustomize: command not found` while running `make deploy`
```

Comment:
This section can probably be safely removed as this issue won't happen if you follow the commands above and errors like "command not found" are pretty self-explanatory. What do you think?

Reply:
Yes, I understand what you mean, my point is that the kustomize and the controller-gen are installed in the  $GOPATH/bin folder, and not in the standard path.

Reply:
That's right, but the $GOBIN should be a part of $PATH

Reply:
All of the tools can be installed in various ways and for different operating systems. This guide is for other developers so we can assume they will manage =)

Reply:
Got you) Done